rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeperServer.java

```diff
@@ -118,4 +127,40 @@ public static boolean isServerUp()
 
     return false;
   }
+
+  /**
+   * Starts a new thread to report the liveness metric at a fixed rate.
+   */
+  private static void scheduleLivenessMetric()
+  {
+    final ScheduledExecutorService metricsService = Executors.newSingleThreadScheduledExecutor();
+    metricsService.scheduleAtFixedRate(new LivenessCheck(), 0, 1000, TimeUnit.MILLISECONDS);
+  }
+
+  /**
+   * Class for emitting a liveness check for the BookKeeper server.
+   */
+  private static class LivenessCheck implements Runnable
+  {
+    @Override
+    public void run()
+    {
+      log.debug("Emitting BookKeeper daemon liveness check");
+      registerLivenessCheck();
+    }
+
+    private void registerLivenessCheck()
+    {
+      if (metrics.getGauges().get(METRIC_BOOKKEEPER_LIVENESS_CHECK) == null) {
+        metrics.register(METRIC_BOOKKEEPER_LIVENESS_CHECK, new Gauge<Integer>()
+        {
+          @Override
+          public Integer getValue()
+          {
+            return 1;
```

Comment:
This is sufficient for now. I want to have a health registry for all components in the bookkeeper like the server and disk monitor. This function should check the health of all the components and then return 1. I'll file an issue for this.